### ISSUE TITLE
Purchased content from new messages is no longer being scraped

### ISSUE THREAD

**Issue by thiccddlp** [DIGITALCRIMINALS/OnlyFans#603]
I have attempted to scrape some content from messages which I purchased yesterday and was sent recently. Older purchased content and messages are still scraping fine however new stuff is being ignored. I believe I am on the latest release of the Scraper, OF snippet of config for reference:

{
  "info": {
    "version": 8.0
  },
  "settings": {
    "auto_site_choice": "",
    "export_type": "json",
    "profile_directories": [
      ".profiles"
    ],
    "max_threads": -1,
    "min_drive_space": 0,
    "helpers": {
      "renamer": true,
      "reformat_media": true,
      "downloader": true,
      "delete_empty_directories": false
    },
    "webhooks": {
      "global_webhooks": [],
      "global_status": true,
      "auth_webhook": {
        "succeeded": {
          "webhooks": [],
          "status": null,
          "hide_sensitive_info": true
        },
        "failed": {
          "webhooks": [],
          "status": null,
          "hide_sensitive_info": true
        }
      },
      "download_webhook": {
        "succeeded": {
          "webhooks": [],
          "status": null,
          "hide_sensitive_info": true
        },
        "failed": {
          "webhooks": [],
          "status": null,
          "hide_sensitive_info": true
        }
      }
    },
    "exit_on_completion": false,
    "infinite_loop": true,
    "loop_timeout": "0",
    "dynamic_rules_link": "https://raw.githubusercontent.com/DATAHOARDERS/dynamic-rules/main/onlyfans.json",
    "proxies": [],
    "cert": "",
    "random_string": "0ccf6416240511edb8a6d8cb8a79e949"
  },
  "supported": {
    "onlyfans": {
      "settings": {
        "auto_profile_choice": [],
        "auto_model_choice": false,
        "auto_api_choice": true,
        "auto_media_choice": "",
        "browser": {
          "auth": true
        },
        "jobs": {
          "scrape": {
            "subscriptions": true,
            "messages": true,
            "paid_content": true
          },
          "metadata": {
            "posts": true,
            "comments": true
          }
        },
        "download_directories": [
          ".sites"
        ],
        "file_directory_format": "{site_name}/{model_username}/{api_type}/{value}/{media_type}",
        "filename_format": "{filename}.{ext}",
        "metadata_directories": [
          ".sites"
        ],
        "metadata_directory_format": "{site_name}/{model_username}/Metadata",
        "delete_legacy_metadata": false,
        "text_length": 255,
        "video_quality": "source",
        "overwrite_files": false,
        "date_format": "%d-%m-%Y",
        "ignored_keywords": [],
        "ignore_type": "",
        "blacklists": "",
        "webhook": true
      }
    }

**Comment by hydr0homiee** [comment]
Yep, been happening for a while now #587 #571 #493

Seems like the fix is hit or miss. It used to work for me but doesn't anymore.

**Comment by hydr0homiee** [comment]
> Same here. Tested with newly-purchased content via messages, no download success regardless of deletion of metadata or the source-model folder itself. Do you feel we need to dig deeper and remove further metadata dbs or anything of that order?
> 
> > Yep, been happening for a while now #587 #571 #493
> > Seems like the fix is hit or miss. It used to work for me but doesn't anymore.

I think it's something on OF's end with how they handle paid content, not anything with the metadata.

I switched back to the `7.6.1` release instead of `master` and followed the hotfix from another comment. I'm able to scrape all free/sent content that's not paid now. But I'm still unable to download any new unlocked paid content. I've tried both the trick with messages/paid content in config and leaving paid content on.

It seems like the scraper stopped downloading paid content some time in June 2022. I'm able to download paid content that I didn't scrape up to the end of May.
